ci/generate-docs.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -e -u -x -o pipefail
+
+if [ ! "$#" -eq 1 ]; then
+    echo "Expected usage: generate-docs.sh <git_hash>"
+    exit 1
+fi
+
+TAG="${1}"
+
+function cleanUp() {
+    # Ensure we are not in the temp dir before cleaning it
+    cd "${CURRENT_DIR}"
+    if [[ -n "${TMP_DIR-}" ]]; then
+        rm -rf "${TMP_DIR}"
+    fi
+}
+
+trap cleanUp EXIT
+
+cd "$(dirname "$0")/../"
+
+ci/bootstrap.sh
+
+# TODO: Remove when merged or moved to other repo.
```

Comment:
Is there a ticket for this work?

Reply:
Its more or less the entire epic - I can probably just move it into its own repo for now